WordPress/WordPressTest/Extensions/StringHelperTests.swift

```diff
@@ -20,6 +20,12 @@ class StringHelperTests: XCTestCase {
         XCTAssert(trimmedString == sourceString.trim())
     }
 
+    func testStripHtmlTags() {
```

Comment:
Great test.

What do you think of adding one to track what happens to HTML entities? This may be a place where the regex and `NSAttributedString` methods differ.